OfficeDev/office-ui-fabric-react
# Fix a11y violations reported in ContextualMenu examples

#### Pull request checklist

- [x] Addresses an existing issue: Fixes #6360

#### Description of changes

This PR addresses the a11y violations for `ContextualMenu` examples. There is no impact to the lib, but they were flagged in #6360 so we opted to address them so folks don't copy invalid code.

**Fixes:**

- Bug 1 : [Accessibility][Forms and validation]: 'Actions' label is not associated with its corresponding control. 
- Bug 2 : [Accessibility][Screen reader]: Label or place holder is not provided for the Edit field(250) present under Contextual Menu with Sub Menus
- Bug 12 : [Accessibility][Programmatic access]: id attribute value must be unique.

`ContextualMenu` component a11y fixes to follow in subsequent PR.

#### Focus areas to test

1. Load the `ContextualMenu` example pages.
1. Verify that id attributes are unique.
1. Verify that inputs have labels or `aria-label` defined.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/6574)
